includes/class-wc-taxjar-integration.php

```diff
@@ -112,397 +103,14 @@ public function __construct() {
-							'class'       => 'input-text disabled regular-input',
-							'id'          => 'woocommerce_taxjar-integration_settings[store_state]',
-						);
-						$settings[] = array(
-							'title'       => __( 'Ship From Postcode / ZIP', 'wc-taxjar' ),
-							'type'        => 'text',
-							'description' => __( 'Enter the zip code from which your store ships products.', 'wc-taxjar' ),
-							'desc_tip'    => true,
-							'default'     => '',
-							'id'          => 'woocommerce_taxjar-integration_settings[store_postcode]',
-						);
-						$settings[] = array(
-							'title'       => __( 'Ship From Country', 'wc-taxjar' ),
-							'type'        => 'hidden',
-							'description' => __( 'We have automatically detected your ship from country as being ', 'wc-taxjar' ) . $store_settings['country'] . '.<br>' . __( 'You can change this setting at', 'wc-taxjar' ) . ' <a href="' . get_admin_url( null, 'admin.php?page=wc-settings' ) . '">WooCommerce -> Settings -> General -> Base Location</a>',
-							'class'       => 'input-text disabled regular-input',
-							'id'          => 'woocommerce_taxjar-integration_settings[store_country]',
-						);
-						$settings[] = array(
-							'type' => 'sectionend',
-						);
-					}
-
-					$settings[] = array(
-						'type' => 'title',
-					);
-					$settings[] = $this->download_orders->get_form_settings_field();
-					$settings[] = array(
-						'title'   => __( 'Debug Log', 'wc-taxjar' ),
-						'type'    => 'checkbox',
-						'default' => 'no',
-						'desc'    => __( 'Log events such as API requests.', 'wc-taxjar' ),
-						'id'      => 'woocommerce_taxjar-integration_settings[debug]',
-					);
-					$settings[] = array(
-						'type' => 'sectionend',
-					);
-					$settings[] = array(
-						'type' => 'title',
-						'desc' => __( 'If you find TaxJar for WooCommerce useful, please rate us <a href="https://wordpress.org/support/plugin/taxjar-simplified-taxes-for-woocommerce/reviews/#new-post" target="_blank">&#9733;&#9733;&#9733;&#9733;&#9733;</a>. Thank you!', 'wc-taxjar' ),
-					);
-					$settings[] = array(
-						'type' => 'sectionend',
-					);
-				}
-			}
-
-			return apply_filters( 'woocommerce_get_settings_' . $this->id, $settings );
-		}
-
-		/**
-		 * Get sections.
-		 *
-		 * @return array
-		 */
-		public function get_sections() {
-			$sections = array(
-				''                     => __( 'Settings', 'woocommerce' ),
-				'transaction_backfill' => __( 'Transaction Backfill', 'wc-taxjar' ),
-				'sync_queue'           => __( 'Sync Queue', 'wc-taxjar' ),
-			);
-			return apply_filters( 'woocommerce_get_sections_' . $this->id, $sections );
-		}
-
-		/**
-		 * Save settings.
+         * Get the regions API endpoint URI
+         *
+		 * @return string
 		 */
-		public function save() {
-			$settings = $this->get_settings();
-			WC_Admin_Settings::save_fields( $settings );
-		}
+		public static function get_regions_uri() {
+		    return self::$app_uri . 'account#states';
+        }
```

Comment:
@dallendalton Do we use the `get_regions_uri` method anywhere? Or is this no longer needed?